src/parallel.h

```diff
@@ -25,38 +25,50 @@ using namespace std;
 
 extern CCriticalSection cs_blockvalidationthread;
 
-// adds all the script check queues into the "allScriptCheckQueues" global variable and creates nScriptCheckThreads per check queue
+// adds all the script check queues into the "allScriptCheckQueues" global variable and creates nScriptCheckThreads per
+// check queue
 void AddAllScriptCheckQueuesAndThreads(int nScriptCheckThreads, boost::thread_group* threadGroup);
 // Entry point for the script check queue threads.
 void AddScriptCheckThreads(int i, CCheckQueue<CScriptCheck>* pqueue);
 
 extern CCriticalSection cs_semPV;
-extern CSemaphore *semPV; // semaphore for parallel validation threads
+extern CSemaphore* semPV; // semaphore for parallel validation threads
```

Comment:
In C++, the * is not part of the type but of the "declarator". This is why

    int *a, b;

Don't have the semantic most people expect. It's better to have the * or the & stick with the declarator rather than the type.